build_tools/bundle.js

```diff
@@ -83,6 +83,18 @@ function aliasDirectories(mapDirectories, bundler) {
     });
 }
 
+function files(patterns) {
+    if (!Array.isArray(patterns)) {
+        return glob.sync(patterns);
+    }
+
+    var fileGroups = patterns.map(function(pattern) {
+        return glob.sync(pattern);
+    });
+
+    return Array.prototype.concat.apply([], fileGroups);
```

Comment:
Assuming you're using `concat.apply([]...` to flatten `fielgroups`, is that necessary?